internal/cobraprompt/prompt.go

```diff
@@ -69,15 +101,23 @@ func (co CobraPrompt) Run(ctx context.Context) {
 			}
 			os.Args = append([]string{os.Args[0]}, promptArgs...)
 			if err := co.RootCmd.ExecuteContext(ctx); err != nil {
+				if err == ExitError {
```

Comment:
Let's use `errors.Is` here too.

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/35/commits/bdd565a3b18e25cbb9e967db21a395523885a7e1